section_to_course/admin.py

```diff
@@ -0,0 +1,357 @@
+"""
+Admin views for creating and managing section-based courses.
+"""
+import json
+
+from django import forms
+from django.conf import settings
+from django.contrib import admin
+from django.contrib.admin.widgets import SELECT2_TRANSLATIONS, AutocompleteSelect
+from django.core import validators
+from django.core.exceptions import ValidationError
+from django.urls import reverse
+from django.utils.html import format_html
+from django.utils.translation import get_language
+from django.utils.translation import gettext as _
+from django_object_actions import DjangoObjectActions
```

Comment:
I've added a new requirements file, local.in, since the requirements for Palm and the requirements for Nutmeg are incompatible and there's no set of requirements which will satisfy them both. I've added it to the README and indicated when it should be used. Will this suffice?

Reply:
@Kelketek, why do you want to avoid using `base.txt`? `local.in` dependencies are the ones that are used only locally. These dependencies will be used both during the development and in the production environment, so their correct place is in `base.in`.

If you are worried about compatibility, then you don't need to specify min versions in `base.txt`. According to `setup.py`, only `base.in` and `constraints.txt` (+ [common_constraints](https://raw.githubusercontent.com/edx/edx-lint/master/edx_lint/files/common_constraints.txt)) are used in the Python package installation process. Constraints should define only upper limits for the package versions. Therefore, there is no risk of re-installing dependencies when installing the `section-to-course` Python package in the `edx-platfom`.

Reply:
@Kelketek, why do you want to avoid using `base.txt`? `local.in` dependencies are the ones that are used only locally. These dependencies will be used both during the development and in the production environment, so their correct place is in `base.in`.

If you are worried about compatibility, then you don't need to specify min versions in `base.txt`. According to `setup.py`, only `base.in` and `constraints.txt` (+ [common_constraints](https://raw.githubusercontent.com/edx/edx-lint/master/edx_lint/files/common_constraints.txt)) are used in the Python package installation process. Constraints should define only upper limits for the package versions. Therefore, there is no risk of re-installing dependencies when installing the `section-to-course` Python package in the `edx-platfom`.

Reply:
@Agrendalath I'll try specifying max versions for each of the dependencies and get back to you. Maybe the issue was coming from me specifying only min.

Reply:
I think that for now we should be fine without specifying any package versions. `common_constraints` is good enough for the current use case. Once we have integration tests working, we can use tox to add more constraints.

Reply:
@Agrendalath I could have sworn I'd tried using no constraints already, but trying it now, it works fine-- both with Nutmeg and with Palm. I guess I didn't and only ever tried with minimum versions.